chart/epinio/questions.yml

```diff
@@ -118,47 +118,48 @@ questions:
   type: boolean
   group: "S3 storage"
   show_subquestion_if: false
-- variable: s3gw.enabled
```

Comment:
I believe that the minio description should be modified as well.

Instead of   `description: "Disable Minio to use s3gw."`
Use `  description: "Disable Minio to use s3gw or external S3."`
To make it plain at the top-level visible description that this is where external S3 is hidden.

Reply:
done